raiden/tests/integration/network/proxies/test_token_network.py

```diff
@@ -788,7 +788,7 @@ def test_token_network_actions_at_pruned_blocks(
     )
 
     # Now wait until this block becomes pruned
-    pruned_number = c1_chain.block_number()
+    pruned_number = c1_chain.client.block_number()
```

Comment:
Changes like this is why I also said it makes sense to leave it as is. We use it a lot and is a nice convenience. While now you have to access the client through the blockchain service adding extra access operators ".".